components/side-bar.tsx

```diff
@@ -94,50 +52,29 @@ const SideBar: FC<SideBarProps> = ({ inSideDrawer = false }) => {
             <>
               <div key={index} className="px-3">
                 {inSideDrawer ? (
-                  <SheetClose
-                    onClick={() => handleOptionClick(index, route.href)}
-                    className={`flex cursor-pointer items-center gap-4 p-2 `}
-                  >
-                    <Image
-                      src={
-                        selectedOption === index ? route.activeIcon : route.icon
-                      }
-                      alt={route.name}
-                      width={24}
-                      height={24}
-                    />
-
-                    <p
-                      className={`${selectedOption === index ? "text-primary" : "text-heading"} flex-1 text-sm font-normal`}
+                  <Link href={route.href}>
+                    <SheetClose
+                      className={`flex cursor-pointer items-center gap-4 p-2 ${pathname === route.href ? "text-primary" : "text-heading"} hover:text-primary`}
                     >
-                      {route.name}
-                    </p>
-                  </SheetClose>
+                      <route.icon />
+
+                      <p className={` flex-1 text-sm `}>{route.name}</p>
+                    </SheetClose>
+                  </Link>
                 ) : (
-                  <div
-                    onClick={() => handleOptionClick(index, route.href)}
-                    className={`flex cursor-pointer items-center gap-4 p-2 `}
+                  <Link
+                    href={route.href}
+                    className={`flex cursor-pointer items-center gap-4 p-2 ${pathname === route.href ? "text-primary" : "text-heading"} hover:text-primary`}
                   >
-                    <Image
-                      src={
-                        selectedOption === index ? route.activeIcon : route.icon
-                      }
-                      alt={route.name}
-                      width={24}
-                      height={24}
-                    />
+                    <route.icon />
 
-                    <p
-                      className={`${selectedOption === index ? "text-primary" : "text-heading"} flex-1 text-sm font-normal`}
-                    >
-                      {route.name}
-                    </p>
-                  </div>
+                    <p className={`flex-1 text-sm`}>{route.name}</p>
+                  </Link>
                 )}
               </div>
 
               {index === 3 && (
-                <hr className="my-4 me-3 ms-1 h-px text-[#E5E9ED]" />
+                <hr className="my-4 me-3 ms-1 h-px text-seperator" />
```

Comment:
text color for `hr` tag ? 
It will be `bg`

Reply:
the line gets the text color not the bg color

Reply:
removed it